app/views/candidate_interface/application_form/review.html.erb

```diff
@@ -7,7 +7,21 @@
 
 <h2 class="govuk-heading-m govuk-!-font-size-27">Courses</h2>
 
-<%= govuk_button_link_to 'TODO: Add a course', '#', class: 'govuk-button--secondary govuk-!-font-size-16 govuk-!-padding-top-1 govuk-!-padding-bottom-1' %>
+<% @application_form.application_choices.map.each do |course_choice| %>
+  <%= render(SummaryCardComponent, rows: [
+    { key: 'Course', value: "#{course_choice.course.name} (#{course_choice.course.code})", change_path: candidate_interface_course_choices_course_path(course_choice.provider.code) },
+    { key: 'Location', value: course_choice.site.name },
+  ]) do %>
+    <header class='app-summary-card__header'>
+      <h3 class='app-summary-card__title'>
+        <%= course_choice.provider.name %>
+      </h3>
+      <div class='app-summary-card__actions'>
+        <%= link_to t('application_form.courses.delete'), candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link' %>
+      </div>
+    </header>
+  <% end %>
+<% end %>
```

Comment:
This could be a component, which would make it consistent with the rest of this file. Non-blocking